### ISSUE TITLE
Question: Would you consider adding the cool koala and cool smilies?

### ISSUE THREAD

**Issue by psyanite** [Tarikul-Islam-Anik/Animated-Fluent-Emojis#23]
First things first I want to thank you for putting this together!!! This is the most complete collection I have found on the internet! Thank you!

I noticed some emojis were missing from the Smilies section like cool cat cool koala heart eyes robot etc.
Would you consider adding them in? 

Cheers,
psyanite

**Comment by Tarikul-Islam-Anik** [comment]
Certainly! Thank you for your kind words and for bringing this to my attention, @psyanite ! 😊

Regarding the missing emojis like cool cat, cool koala, heart eyes, robot, etc., I'll definitely look into adding them to the Smilies section for a more comprehensive collection.

In order to ensure I add the emojis you mentioned correctly, could you provide me with an example or specific Unicode representations (if available) for the emojis you mentioned? This will help me accurately include them in the collection.

Once again, thank you for your feedback and for using my project! I'll work on incorporating these emojis to enhance the collection further.

**Comment by psyanite** [comment]
Oh what a wonderful suprise to see you reply so soon! 😄 

Hmm maybe there isn't an unicode? But they are present in the Microsoft page and in the app

https://support.microsoft.com/en-us/office/smilies-in-microsoft-teams-free-56460aca-5d27-4609-a186-575382fdf3ec

<img width="461" alt="Screenshot 2023-11-15 at 3 49 35 pm" src="https://github.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/assets/24215698/79fd51d8-b779-4ebf-9972-ffa4d84177c2">
<img width="451" alt="Screenshot 2023-11-15 at 3 49 51 pm" src="https://github.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/assets/24215698/e2b9793c-6728-46e1-b928-d86e96d3dbdb">

**Comment by Tarikul-Islam-Anik** [comment]
Thank you for sharing a image of the emojis. It appears that the emojis you mentioned might not have Unicode representations. However, I'm actively searching various sources to find these emojis.

During this search process, I've actually come across some new emojis that I'm excited to push to the repository today! I'll continue my efforts to locate the specific emojis you mentioned. If I'm unable to find them through standard Unicode sources, I'll explore the possibility of extracting them from Microsoft Teams.

Rest assured, I'm committed to enhancing the collection and will do my best to include these emojis. Thank you for your patience and understanding! I'm going the keep the issue open and I'll keep you informed about any progress or updates.

Have a great day!

**Comment by psyanite** [comment]
Oh! Yeah it can't be helped these are microsoft custom ones, even though they're so adorable.
Oh I'm so excited for the new emojis!!! Looking forward to it 🔥

**Comment by adrianensan** [comment]
Not sure if this helps, but here's pretty much all the non-unicode ones, along with a few that don't seem to be animated in this repo, like melting face. Unfortunately, these are only 100x100px, but better than nothing until higher res versions are found
[emojis.zip](https://github.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/files/13443415/emojis.zip)

**Comment by psyanite** [comment]
@adrianensan ! nice job! I'm going to upload some of these to my work slack right away! thansk a bunch
